packages/core/src/node/bunyan-logger-server.ts

```diff
@@ -42,11 +45,9 @@ export class BunyanLoggerServer implements ILoggerServer {
         if (['trace', 'debug', 'info', 'warn', 'error', 'fatal'].indexOf(logLevel) < 0) {
             logLevel = 'info';
         }
-
-        this.loggers.push(bunyan.createLogger({
-            name: 'Theia',
-            level: logLevel
-        }));
+        this.loggers.push(bunyan.createLogger(
+            JSON.parse(JSON.stringify(serverOptions))
+        ));
```

Comment:
So this takes an object, makes it a string and then makes it an object again...? I fail to see the point